include/fmt/std.h

```diff
@@ -73,6 +73,7 @@ inline void write_escaped_path<std::filesystem::path::value_type>(
 }
 
 }  // namespace detail
+FMT_BEGIN_EXPORT
```

Comment:
Here and a few other places: no need for a block, it's enough to mark the single definition here with `FMT_MODULE_EXPORT`.